libs/SalesforceNetwork/SalesforceNetwork/SalesforceNetwork/Classes/Network/Action/CSFAction.h

```diff
@@ -294,6 +296,6 @@ CSF_EXTERN NSString * const kCSFActionTimingPostProcessingKey;
  
  @return The time interval for the requested key, or `0` if the key is invalid or if the request hasn't gathered enough information to supply that value yet.
  */
-- (NSTimeInterval)intervalForTimingKey:(NSString*)key;
+- (NSTimeInterval)intervalForTimingKey:(CSFActionKey)key;
```

Comment:
Nope, if this was C++ it might have to be but objective-C is weakly typed and typedef is basically just a macro for type replacement. That enum macro is basically riding on top of that to work. For consistency I can change it there too if you would like.